run/markdown-preview/editor/test/service-name.sh

```diff
@@ -0,0 +1,23 @@
+#!/usr/bin/env bash
+
+# Copyright 2020 Google LLC.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -eo pipefail;
```

Comment:
Let's add this to the main file since separating off this small bit of code doesn't add any value. You are also defining UPSTREAM_SAMPLE_NAME in multiple locations across this file and other files.

Reply:
Ok, done. I also removed the UPSTREAM_SAMPLE_NAME var since it was hardcoded to "renderer", so I inserted "renderer" wherever the variable was used.